resources/views/components/sidebar/index.blade.php

```diff
@@ -1,3 +1,3 @@
-<div class="space-y-5 md:mt-[52px] max-md:mt-10 md:[&>*]:w-[370px] md:[&>*]:ml-9">
+<div class="mt-10 space-y-5 md:mt-[50px]">
```

Comment:
By removing those arbitrary values, does the the main layout still goes fullscreen if there is not content within the sidebar? can you check that for me?

Reply:
yes it does